README.rst

```diff
@@ -55,6 +55,37 @@ Dev mode
 
    pip install -e .[dev]
 
+Version control of database using DVC (Data Version Control)
+------------
```

Comment:
I guess the underline heading marker has to go all the way to the end of the text.

(This heading is kind of long though.)

```suggestion
------------------------------------------------------------
```